app/main.py

```diff
@@ -2,0 +1,14 @@
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] != "cp":
+        print("Invalid command format.")
+        return
+    first_txt, second_txt = command_parts[1], command_parts[2]
+    if first_txt == second_txt:
+        print("Source and destination files are the same.")
+        return
+    else:
+        with (open(first_txt, "r") as file_in,
+              open(second_txt, "w") as file_out):
```

Comment:
Clean Code: The `with` statement is not correctly used. In Python, you cannot open two files in one `with` statement like this. You should use a separate `with` statement for each file.